state/store.go

```diff
@@ -137,10 +137,10 @@ func (arz *ABCIResponses) ResultsHash() []byte {
 // LoadABCIResponses loads the ABCIResponses for the given height from the database.
 // This is useful for recovering from crashes where we called app.Commit and before we called
 // s.Save(). It can also be used to produce Merkle proofs of the result of txs.
-func LoadABCIResponses(db dbm.DB, height int64) (*ABCIResponses, error) {
-	buf := db.Get(calcABCIResponsesKey(height))
+func LoadABCIResponses(db dbm.DB) (*ABCIResponses, error) {
```

Comment:
The comment above is out dated now: "LoadABCIResponses loads the ABCIResponses for the given height"